jenkins/development.Dockerfile.j2

```diff
@@ -40,17 +42,25 @@
 #
 # [1] See https://www.redhat.com/sysadmin/user-flag-rootless-containers
 
-FROM quay.io/fedora/fedora:33
+FROM {{ base_image_ref }}
 
 RUN \
-    dnf install -y \
+{%- if hosts_file_url %}
+    curl -sSk -o /etc/build.hosts {{ hosts_file_url }} && \
+    cat /etc/build.hosts >> /etc/hosts && \
+{% endif %}
+{%- if repo_file_url %}
+    curl -sSk -o /etc/yum.repos.d/build.repo {{ repo_file_url }} && \
+{% endif %}
```

Comment:
Intriguing; a comment to justify/explain this would be interesting. I assume there's a use case for this being used in our Jenkins fleet?

Reply:
This is not used by our Jenkins fleet.  This is used by a pbench-server-in-a-can, since a UBI base doesn't provide all the RPMs and python modules we need to build it.